### ISSUE TITLE
casecode processes multiple names in a strange way

### ISSUE THREAD

**Issue by aplteam** [the-carlisle-group/Acre-Desktop#157]
According to the documentation only the first name should be processed.

This is what actually happens:

```
      ⎕SE.acreAPI.casecode'#.TEMP.Foo #.TEMP.Fooü'
#.TEMP-f.Foo #-1.TEMP-f.Fooü-1
```

**Comment by PhilLast** [comment]
It _is_ a bug because in the above case it should actually return an empty result and an error message because acre nowhere handles single strings containing multiple names.

However:
```
      ⎕SE.acreAPI.casecode'#.TEMP.Foo' '#.TEMP.Fooü'
#.TEMP-f.Foo-1
```

**Comment by PhilLast** [comment]
To clarify my statement above
> nowhere handles single strings containing multiple names

by the time the acre function gets the argument from the user command interface it has been parsed into 'separate' 'words' and deliberately "quoted strings". Perhaps the `⎕SE.acreAPI` interface should be capable of the same operation. Easiest would be to reject them as invalid.

**Comment by PhilLast** [comment]
Fixed by commit `Acre 5.0.8+148 2019-03-15 15.25`
